### ISSUE TITLE
Virtualbox is not compatible

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#26]
```
It does not work correctly in Virtualbox. 
Similar to #4 (Virtualbox is QT application)

Example:
Hallo -> hallo or halllo
The result is random.

My workaround:
#script
for x in 'Hallo':
    system.exec_command("sleep 0.01 && xdotool type " + x)
(xdotool must be installed)


version:
0.61.5 on Ubuntu 9.10



```

Original issue reported on code.google.com by `pisztra...@gmail.com` on 23 Mar 2010 at 10:10

**Comment by GoogleCodeExporter** [comment]
```
This is a bug in QT. It is fixed in QT4.5, hopefully one day Sun will finally 
get
their act together and upgrade Virtualbox to use it.
```

Original comment by `cdekter` on 24 Mar 2010 at 11:47
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Nasty workaround 'fix' will be in v0.80.0 as Virtualbox still exhibits this 
problem after more than a year.
```

Original comment by `cdekter` on 19 Sep 2011 at 1:33
- Changed state: **Fixed**
